libs/components/code-examples/src/lib/modules/forms/radio/standard/example.component.html

```diff
@@ -1,29 +1,27 @@
-<div class="sky-padding-even-lg">
```

Comment:
Removing these wrappers so that the demos don't have excessive padding on the docs site.